### PR TITLE
Higher order Functions Version of 2048

### PR DESCRIPTION
**Pull request by colonlc**
See readme in my [repository](https://github.com/colonlc/CC_2048).
Proposed solution to my comment on video number 4 :).

### CODE REVIEW

File: _CodingChallenges/94.4-2048.md [hunk 1]
```diff
@@ -22,7 +22,14 @@ parts:
   url: "/CodingChallenges/94.2-2048"
 - title: "2048 - Part 3"
   url: "/CodingChallenges/94.3-2048"
-
+  
+contributions:
+  - title: "Funcional with higher order functions Version of 2048"
```
Comment (shiffman) [inline _CodingChallenges/94.4-2048.md]: A little typo here in "Functional"!

**Comment by colonlc** [comment]
Fixed the typo.

**Comment by meiamsome** [comment]
Hey, we made a change so community contributions are only acceptable in the 1st file of a multi-part Coding Challenge, so we need to move that contribution to that file. It would be great if you could do that, but if not I'll do it in a bit

**Comment by colonlc** [comment]
moved it to part one :)

**Comment by meiamsome** [comment]
Thanks for this, and for your patience.

**Pull request merged**